scripts/src/main/resources/scripts/command/vscode

```diff
@@ -122,10 +122,55 @@ function doRunVsCode() {
 
 function doStartVsCode() {
   doConfigureVsCode -u
+  doAddRecmdt
   doEcho "launching VSCode..."
   doRunVsCode "${WORKSPACE_PATH}"
 }
 
+function doAddRecmdt() {
+  local recmdtJSON="${WORKSPACE_PATH}/.vscode/extensions.json"
+  local file
+  local recmdtKey
+  local recmdtValue
+  local pluginsToRecmdt=()
+  for file in "${SETTINGS_PATH}"/vscode/plugins/*.properties
+  do
+    if [ -f "${file}" ]
+    then
+      plugin_id=""
+      plugin_active="true"
+      doLoadProperties "${file}"
+      if [ -z "${plugin_id}" ]
+      then
+        doWarning "Invalid vscode plugin config: ${file}"
+      elif [ "${plugin_active}" = "false" ]
+      then
+        local IFS=, 
+        pluginsToRecmdt+=("\"${plugin_id}\"")
+      fi
+    fi
+  done
+  recmdtValue=${pluginsToRecmdt[*]}
+  recmdtKey="\"recommendations\": [$recmdtValue]"
+  if [ -f "${recmdtJSON}" ] && [ ! -s "${recmdtJSON}" ]
+  then 
+    if grep -q "\"recommendations\":" "${recmdtJSON}"
+    then
+      if ! grep -Fq "${recmdtKey}" "${recmdtJSON}"
+      then
+        sed -i "s/\"recommendations\": \[[^]]*\]/${recmdtKey}/g" "${recmdtJSON}"
```

Comment:
I had some difficulties adding entries in a JSON property. There is a tool `jq`, but since this is not a default tool, I chose some `sed` commands. If there are better solutions, I am happy to hear that.

Reply:
Correct. I also used `jq` once in a different context but as you already wrote we can not expect it to be installed.
We would need to add a commandlet `jq` that download and installs it in order to use it here. Seems overkill as long as you can solve it with `sed` hacking :)